test/dlc_tests/sanity/test_security_check.py

```diff
@@ -101,11 +101,20 @@ def test_ecr_scan(image, ecr_client, sts_client, region):
     """
     test_account_id = sts_client.get_caller_identity().get("Account")
     image_account_id = get_account_id_from_image_uri(image)
+    image_repo_uri, image_tag = image.split(":")
+    _, image_repo_name = image_repo_uri.split("/")
```

Comment:
Please use existing utility functions such as `get_repository_and_tag_from_image_uri` to obtain the `image_repo_name` and `image_tag`, or create a new utility function to get the `image_repo_uri` from an ECR URI, rather than string-parsing right here.